.github/workflows/examples.yml

```diff
@@ -0,0 +1,75 @@
+name: Examples
+
+on:
+  # Trigger the workflow on push or pull request,
+  # but only for the master branch
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  tutorials:
+    name: Examples with conda install
+    runs-on: ubuntu-latest
+
+    env:
+      DEVITO_ARCH: gcc-7
+      DEVITO_OPENMP: 1
+      DEVITO_BACKEND: "core"
+      PYTHON_VERSION: "3.7"
+
+    strategy:
+      # Prevent all build to stop if a single one fails
+      fail-fast: false
+
+    steps:
+    - name: Checkout devito
+      uses: actions/checkout@v1
+
+    - name: Setup conda
+      uses: goanpeca/setup-miniconda@v1
+      with:
+       activate-environment: devito
+       environment-file: environment.yml
+       python-version: 3.7
+       auto-activate-base: false
+
+    - name: Install dependencies
+      # See setup-miniconda README, necessary to activate environment
+      shell: bash -l {0}
+      run: |
+        pip install -e .
+        pip install matplotlib
```

Comment:
actually necessary?

Reply:
yes, matplotlib is in optionals but needed by the cfd test/example